RomM/romm.py

```diff
@@ -352,6 +358,17 @@ def _update_roms_view(self):
             self.status.current_filter = next(self.status.filters)
             self.roms_selected_position = 0
             self.input.reset_input()
+        elif self.input.key("R1"):
+            if len(self.status.multi_selected_roms):
+                self.status.multi_selected_roms = []
+            else:
+                self.status.multi_selected_roms = []
+                for r in self.status.roms_to_show:
+                    self.status.multi_selected_roms.append(
+                        r
+                    )
```

Comment:
Does this mean select all only select the 10-ish games shown in the screen, instead of the entire library? Not that I'm saying it should btw.

Reply:
Cool going pull this branch and give it a shot 👍🏼